### DISCUSSION TITLE
Closable tabs

### DISCUSSION THREAD

**Discussion by eduardoghi** [saadeghi/daisyui#2779] [Ideas / feature requests]
It's possible to add close buttons on tabs and have a way to disable/enable the button?

Example:
![image](https://github.com/saadeghi/daisyui/assets/124214754/d952bbb6-aee2-4e92-b7c9-32e4f282b1b6)

**Comment by saadeghi** [comment]
It can't be done with [radio tabs](https://daisyui.com/components/tab/#radio-tab-bordered--tab-content) but it's possible with a <button> with tab class. You can put an icon in it and remove it using JS when it's clicked.

Let me know if you have a question.
